metrics/src/main/java/com/google/gson/metrics/ParseBenchmark.java

```diff
@@ -107,13 +107,13 @@ private Document(TypeToken<?> typeToken, TypeReference<?> typeReference) {
     parser = api.newParser();
   }
 
-  public void timeParse(int reps) throws Exception {
+  public void timeParse(final int reps) throws Exception {
```

Comment:
We don't mark parameters as final since it's prohibitively noisey. IntelliJ IDEA 2018.2 will highlight locals and parameters differently depending on whether you overwrite them or not and we think that's good enough.